### PR TITLE
docs(dotnet): document unsupported Parallelizable attributes on Nunit

### PR DESCRIPTION
**Pull request by kblok**
Related to https://github.com/microsoft/playwright-dotnet/issues/

I'm clarifying what the valid options for `Parallelizable` are.

### CODE REVIEW

File: docs/src/test-runners-csharp.md [hunk 1]
```diff
@@ -58,6 +60,10 @@ public class MyTest : PageTest
 }
 ```
 
+:::note
+You can only set Parallelizable to ParallelScope.Self. ParallelScope.All and ParallelScope.Fixtures are not supported.
```
Comment (mxschmitt) [inline docs/src/test-runners-csharp.md:64]:
```suggestion
You can only set Parallelizable to `ParallelScope.Self`. `ParallelScope.All` and `ParallelScope.Fixtures` are not supported.
```

**Comment by mxschmitt** [comment]
Can we add a similar section for MSTest?

**Comment by kblok** [comment]
Done @mxschmitt

File: docs/src/test-runners-csharp.md [hunk 2]
```diff
@@ -58,6 +60,10 @@ public class MyTest : PageTest
 }
 ```
 
+:::note
```
Comment (mxschmitt) [inline docs/src/test-runners-csharp.md:63]: can we move it under the "### Running NUnit tests in Parallel" heading and don't have a Note for it?

Rest lgtm.

Comment (kblok) [reply docs/src/test-runners-csharp.md]: How about now @mxschmitt ?

**Review by mxschmitt — APPROVED** [review]

File: docs/src/test-runners-csharp.md (hunk 2 above)
Comment (mxschmitt) [reply docs/src/test-runners-csharp.md]: amazing!

**Pull request merged**